d/dracut/stone.yaml

```diff
@@ -24,6 +24,12 @@ rundeps     :
     - binary(cpio)
     - terminus-font
 setup       : |
+    %patch %(pkgdir)/patches/downstream/0001-Add-module-to-invalidate-depmod-index.patch
```

Comment:
This seems Solus specific not sure where/how it integrates into Serpent atm. Any upstream tracking bug for this in dracut-ng?

Reply:
This provides the functionality that we talked about where depmod is ran in the initrd when an initrd extension is present with kernel modules. Without this the addon kernel modules will not be available which defeats the purpose of having initrd modules in the first place.